composer.json

```diff
@@ -47,5 +47,8 @@
     "psr-4": {
       "PerformanceLab\\Tests\\": "tests/utils"
     }
+  },
+  "autoload": {
+    "classmap": ["includes/"]
```

Comment:
See my other comment on autoloading, please revert this change.

Reply:
This has been removed entirely @felixarntz